app/main.py

```diff
@@ -1 +1,71 @@
 # write your code here
+class SoftwareEngineer:
+    def __init__(
+            self,
+            name: str,
+    ) -> None:
```

Comment:
Isn’t the length of line match flake8 rules? I believe you can put it in one line